exchanges/zb/zb.go

```diff
@@ -312,40 +315,38 @@ func (z *ZB) SendAuthenticatedHTTPRequest(ep exchange.URL, httpMethod string, pa
 		[]byte(params.Encode()),
 		[]byte(crypto.Sha1ToHex(z.API.Credentials.Secret)))
 
-	now := time.Now()
-	params.Set("reqTime", fmt.Sprintf("%d", convert.UnixMillis(now)))
-	params.Set("sign", fmt.Sprintf("%x", hmac))
-
-	urlPath := fmt.Sprintf("%s/%s?%s",
-		endpoint,
-		params.Get("method"),
-		params.Encode())
-
 	var intermediary json.RawMessage
+	newRequest := func() (*request.Item, error) {
+		now := time.Now()
+		params.Set("reqTime", fmt.Sprintf("%d", convert.UnixMillis(now)))
+		params.Set("sign", fmt.Sprintf("%x", hmac))
+
+		urlPath := fmt.Sprintf("%s/%s?%s",
+			endpoint,
+			params.Get("method"),
+			params.Encode())
+
+		return &request.Item{
+			Method:        httpMethod,
+			Path:          urlPath,
```

Comment:
The API url now appears to be `http://api.zb.com` now, not `http://api.zb.live`. This is undocumented however. But I do get successful responses from there